### DISCUSSION TITLE
Dynamic record label addition (i.e. `(Monstercat Release)`) to applicable song title.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#26361] [Ideas]
This is essentially a request to grandfather over the automatic `(Monstercat Release)` from stable to this client through some means.

**Comment by bdach** [comment]
what does this mean

**Comment by Coppertine** [comment]
![screenshot1539](https://github.com/ppy/osu/assets/37494321/b962b448-f1d6-4d93-9be3-9f77d3e172ff)
On certain select Featured Aritst listings, the game intentionally adds `(Monstercat Release)` to the end of the song title / difficulty name. This was implimented soon after Monstercat became a Featured Artist (iirc).
